templates/post_body_generic.html

```diff
@@ -0,0 +1,45 @@
+{{define "post_body_generic"}}
```

Comment:
This is the same as `post_body_full` except that it doesn't include the notification contents